### PR TITLE
DRB PW test failure fix

### PR DESCRIPTION
**Pull request by ayan1229**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1839)

### This PR does the following:
- fixes the automated test failures 
- changes the QA login credentials

### CODE REVIEW

File: playwright/features/pub_yearFilter.feature [hunk 1]
```diff
@@ -8,12 +8,3 @@ Feature: Publication Year Filter
...
         Then the "first search result edition" should be displayed
-
-    Scenario: As a user I filter by publication years and verify the search results
-        Given I go to the "home" page
-        When I fill in the "homepage search box" with "Jane Austen"
-        And I click the "search button"
-        And I fill in the "publication year from filter" with "1800"
-        And I fill in the "publication year to filter" with "1835"
-        And I click the "publication year apply button"
-        Then only editions published between year "1800" and "1835" should be displayed
```
Comment (samanthaandrews) [inline playwright/features/pub_yearFilter.feature:19]: Why is this test being removed? Seems like the scenario itself is still worth testing?

**Review by samanthaandrews — COMMENTED** [review]
Do you need me to update the credentials in github secrets? Happy to do so, just lmk what they are.

File: playwright/features/pub_yearFilter.feature (hunk 1 above)
Comment (ayan1229) [reply playwright/features/pub_yearFilter.feature:19]: There's a scenario for that under publication.feature @samanthaandrews

**Comment by ayan1229** [comment]
> Do you need me to update the credentials in github secrets? Happy to do so, just lmk what they are.

yes please.

```
CATALOG_USERNAME=qatesterDRB
CATALOG_USER_PIN=Nypl123!
```

thank you!

**Comment by samanthaandrews** [comment]
Thank you for adding the jira ticket and description of changes to the PR! It is also helpful to prefix the title with the ticket number, i.e. `SFR-1839: DRB PW test failure fix`. Makes it easier to review past commits and know what changed.

**Review by samanthaandrews — APPROVED** [review]

**Review by jackiequach — APPROVED** [review]
Looks good

**Pull request merged**